libraries/libraries/settings/base.py

```diff
@@ -93,16 +99,6 @@
 WSGI_APPLICATION = 'libraries.wsgi.application'
 
 
-# Database
-# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
-
-DATABASES = {
```

Comment:
so with no db settings in base.py, on my local machine I'd want to insert this stanza into local.py right? Just checking, I think this is ultimately the best way to do things.